Assets/Scripts/Managers/AudioManager.cs

```diff
@@ -0,0 +1,77 @@
+using UnityEngine.Audio;
+using System;
+using UnityEngine;
+
+
+public class AudioManager : Singleton<AudioManager>
+{
+    public static AudioManager Instance;
+    public AudioMixerGroup musicMixerGroup;
+    public AudioMixerGroup sfxMixerGroup;
+    public Sound[] sounds;
```

Comment:
Recommend that this is private and to get all of the sounds with [`FindObjectsOfType`](https://docs.unity3d.com/ScriptReference/Object.FindObjectsOfType.html). Alternatively an `AddSound()` function may be necessary so that additional `Sound`s can be registered if they are instantiated into the scene at a later time. The function should likely be called from the `Sound` script's awake.